docs/views.rst

```diff
@@ -111,9 +111,9 @@ information::
 Method Based Dispatching
 ------------------------
 
-For RESTful APIs it's especially helpful to execute a different function
+For RESTful APIs it's especially helpful to execute a different method
```

Comment:
The change below is good, but I would remove the change on this line. I think it's clearer as-is. (And methods are a type of function, so it's not incorrect as-is.)

Reply:
Sounds good, will revert it. I was also a little hesitant to change this line.

Reply:
Updated.